### PR TITLE
Routes

### PR DESCRIPTION
**Pull request by noeladd**
Ready to merge routes

### CODE REVIEW

File: server/app/routes/resources/index.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+router.get('/:id', function(req, res, next) {
+    Resource.findById(req.params.id)
+    .then(function(resource) {
+        res.json(resource);
+    })
+    .catch(next);
+});
+
+router.get('/type/:type', function(req, res, next) {
```
Comment (glebec) [inline server/app/routes/resources/index.js:25]: This will never trigger because you already have a GET :id route above.

**Comment by glebec** [comment]
@ingridcronburg @noeladd see above

File: server/app/routes/resources/index.js (hunk 1 above)
Comment (glebec) [reply server/app/routes/resources/index.js:25]: Also if you want to get by type, that should be a query string, not a route.

Comment (ingridlarsongreen) [reply server/app/routes/resources/index.js:25]: @glebec Thanks! We're fixing those now.

**Pull request merged**